src/fmu/sumo/explorer/_document_collection.py

```diff
@@ -94,15 +86,11 @@ def aggregate(self, operations):
         query = {**self.query, "size": self.result_count}
         result = self.sumo.post("/search", json=query)
 
-        object_ids = list(map(
-            lambda s : s["_id"],
-            result.json()["hits"]["hits"]
-        ))
+        object_ids = list(map(lambda s: s["_id"], result.json()["hits"]["hits"]))
 
-        response = self.sumo.post("/aggregate", json={
-            "operation": operation_list,
-            "object_ids": object_ids
-        })
+        response = self.sumo.post(
```

Comment:
Should the `put`, `post` etc methods be private to the module and/or class? I guess if someone uses these directly, the explorer is obsolete (rather use the wrapper directly?

Reply:
We need Ådne for this discussion..